### ISSUE TITLE
A11y_.NET Core_WinForms_Remaining_Controls_Testing_Remaining Controls_Non-Text Contrast: Color contrast ratio of the "Track slider" is 1.53:1 which is less than 3:1.

### ISSUE THREAD

**Issue by HindujaArugula** [dotnet/winforms#9259]
### .NET version

.NET-SDK-8.0.100-preview.4.23260.5

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No response

### GitHubTags:
#A11yMAS;#A11yTCS;#BM_.NET Core[Winforms]-Win32-May2023;#.NET Core;#A11ySev2;#WCAG1.4.11;#DesktopApp;#Win11;#NonBenchmark;#Non-TextContrast

### Actual:
Color contrast ratio of the "Track slider" is 1.53:1 which is less than 3:1 which is present beside "Settings for DomainupDown".

### Expected:
Color contrast ratio of the "Track slider" should be more or equal to 3:1 which is present beside "Settings for DomainupDown".

### User Impact:
Low vision users will face difficulty if the color contrast ratio of the "Track Slider" is less than 3:1.

### Attachments:
![NonTextContrast](https://github.com/dotnet/winforms/assets/125462007/a7cf90ef-44ca-46d7-b5c0-2d54dd8b2346)

### Steps to reproduce

1. Install and Open "Winforms" App.
2. "Main Form" screen would appear.
3. TAB to "Remaining_Controls_Testing" and press ENTER key.
4. TAB to "Track Slider".
5. Open "Accessibility Insights for Windows".
6. Now observe that Color contrast ratio of the "Track slider" is 1.53:1 which is less than 3:1 which is present beside "Settings for DomainupDown".

**Comment by merriemcgaw** [comment]
There is no public API from Common Controls for us to be able to specify it. So this is external to Windows and should go under a blanket Common Controls Exception.

**Comment by HindujaArugula** [comment]
As this is a non-blocking issue closing the bug.
GitHubTags:#Closed;#Regressed:06-16-23;
